schema_editor/bower.json

```diff
@@ -11,7 +11,7 @@
     "angular-spinkit": "^0.3.3",
     "angular-resource": "^1.3.0",
     "angular-ui-router": "~0.2.14",
-    "json-editor": "0.7.18",
+    "json-editor": "0.7.21",
```

Comment:
This needs to be kept at 0.7.18, since versions past that don't support spaces in template variable names (which we rely on). I have an open PR with json-editor to address this, but it hasn't been merged.